test/SlowTests/Server/Documents/ETL/Raven/RavenDB_11157_Raven.cs

```diff
@@ -965,10 +965,9 @@ public void Error_if_load_counter_behavior_func_doesnt_match_any_collection_that
 
             config.Initialize(new RavenConnectionString() { Database = "Foo", TopologyDiscoveryUrls = new[] { "http://localhost:8080" } });
 
-            List<string> errors;
-            config.Validate(out errors);
+            config.Validate(out List<string> errors);
 
-            Assert.Equal(1, errors.Count);
+            Assert.True(0 < errors.Count);
```

Comment:
Why can we have more that 1 error?

Reply:
Yes, I added a check for testing if there is `loadCountersOf<CollectionName>Behavior` so there is a load to this collection

Reply:
So how many errors do we expect? 2 ? If then then why not  `Assert.Equal(2, ... `?

Reply:
I don't like tests that test the implementation and demand update on implementation changes.
If in the future someone will decide to add a check he will need to update this test again.
But if you prefer I will use `Equal` here.